src/main/java/com/example/webapp/Controller/EmployeeController.java

```diff
@@ -0,0 +1,57 @@
+package com.example.webapp.Controller;
+
+import com.example.webapp.Service.EmployeeService;
+import com.example.webapp.dto.EmployeeDTO;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/employees")
```

Comment:
You can have an util class and add "/employees" as constant also other static methods or const

Reply:
Ok